src/components/MissionsList.jsx

```diff
@@ -8,10 +8,9 @@ import '../styles/MissionsList.css';
 function MissionsList() {
   const { missions } = useSelector((state) => state.missions);
   const dispatch = useDispatch();
-
   useEffect(() => {
     if (missions.length === 0) dispatch(fetchMissions());
-  }, [missions, dispatch]);
+  }, []);
```

Comment:
Your use effect is working correctly great job. However it I missing the dependencies `dispatch` and `missions.length`.
Don't worry I'll update it in my branch 👍